primer_designer/views.py

```diff
@@ -195,8 +205,8 @@ def create(request, regions_form):
         for pdf in out_pdfs:
             zip_file.write(pdf, Path(pdf).name)
 
-    # delete output dir from tmp/
-    subprocess.run(f'rm -r {out_dir}', shell=True, check=True)
+    # delete output dir from /static/tmp
+    subprocess.run(f'rm -rf {out_dir}', shell=True, check=True)
```

Comment:
what is this deleting?

Reply:
the temporarily-created directory for generated PDFs. Basically below directory
`outname = f'{time_stamp()}-{random_string()}'`

Reply:
ahh cool thats fine, forgot it did that